spec/format_parser_spec.rb

```diff
@@ -177,14 +177,16 @@
     it 'sorts the parsers by priority and name' do
       parsers = FormatParser.parsers_for(
         [:audio, :image],
-        [:cr2, :dpx, :fdx, :flac, :gif, :jpg, :mov, :mp4, :m4a, :mp3, :mpg, :mpeg, :ogg, :png, :tif, :wav]
+        [:cr2, :cr3, :dpx, :fdx, :flac, :gif, :jpg, :mov, :mp4, :m4a, :mp3, :mpg, :mpeg, :ogg, :png, :tif, :wav]
       )
 
       expect(parsers.map { |parser| parser.class.name }).to eq([
         'FormatParser::GIFParser',
         'Class',
         'FormatParser::PNGParser',
+        'FormatParser::MOOVParser',
```

Comment:
Curious, why wasn't this parser here before? 🤔

Reply:
It [was](https://github.com/WeTransfer/format_parser/pull/222/files/f08a399d1d97d5674e146232c542608b8b7130aa#diff-9a6e94baf6d000b5721f4990cf2d55457d60c94614851c6fdcc83dc455da772cL164) only being registered with `:video` as a nature, but that's not representative since it returns `Audio` for M4A files, so I a fixed that up.